staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -477,7 +478,22 @@ func filterByNamespace(objs map[types.NamespacedName]runtime.Object, ns string)
 		if ns != "" && acc.GetNamespace() != ns {
 			continue
 		}
-		res = append(res, obj)
+		switch acc.(type) {
```

Comment:
knowledge of specific types and translations between them do not belong in this file

if fake client is going to be taught about translating an eviction subresource create into a pod delete, it should be done somewhere like staging/src/k8s.io/client-go/kubernetes/typed/core/v1/fake/fake_pod_expansion.go

Reply:
ok thanks for the reminder, I've tried making the changes and resubmitting